recipe/meta.yaml

```diff
@@ -10,9 +10,9 @@ source:
   sha256: f880eada0b8b1f524d5951dc6fcae0d13b169897fc8a247d75fb5beadd69c5f0
 
 build:
-    number: 1
-    skip: True  # [py2k]
-    script: python -m pip install --no-deps --ignore-installed .
+    number: 2
+    noarch: python
+    script: python -m pip install --no-deps .
```

Comment:
Using `--ignore-installed` should be fine, no need to remove it :wink:

Reply:
Ha, that was me cargo-culting from a recipe that was working...

Reply:
Haha thanks, "cargo-culting", yet another new phrase I never heard before -- I'll try to remember it :laughing:

Reply:
Ha, thanks for the review!